internal/compiler/generator/rust.rs

```diff
@@ -114,6 +114,7 @@ fn rust_primitive_type(ty: &Type) -> Option<proc_macro2::TokenStream> {
 fn rust_property_type(ty: &Type) -> Option<proc_macro2::TokenStream> {
     match ty {
         Type::LogicalLength => Some(quote!(sp::LogicalLength)),
+        Type::Easing => Some(quote!(sp::EasingCurve)),
```

Comment:
Is that an unrelated change? (I don't mind it being here, just curious)

Reply:
The test add a `property <easing>` which fails.

The gallery also have that. But in the gallery, the property is optimized out so the rust code compile, but not here.